x-pack/plugins/security_solution/public/timelines/components/open_timeline/translations.ts

```diff
@@ -305,14 +324,14 @@ export const FILTER_CUSTOM_TIMELINES = i18n.translate(
 export const IMPORT_TIMELINE_BTN_TITLE = i18n.translate(
   'xpack.securitySolution.timelines.components.importTimelineModal.importTimelineTitle',
   {
-    defaultMessage: 'Import timeline',
+    defaultMessage: 'Import',
   }
 );
 
 export const SELECT_TIMELINE = i18n.translate(
   'xpack.securitySolution.timelines.components.importTimelineModal.selectTimelineDescription',
   {
-    defaultMessage: 'Select a Security timeline (as exported from the Timeline view) to import',
+    defaultMessage: 'Select a security timeline or timeline template file to import',
```

Comment:
>  (copy help needed)
> Note: As we can import multiple timelines / templates in a file at a time, so I change the wording to Select a security timeline or timeline template file to import instead

The following text incorporates the multiple selection feature noted above:

`Select one or more timeline or timeline template files to import`

The word `security` is omitted in the suggestion above, which leaves open the possibility to import timelines or templates in an non-security context in the future.

Reply:
Yup, omitted security sounds good, but at the moment we still only support importing a single ndjson file at a time. (However the ndjson file might contain more than one timeline or timeline template's json object in it.)
So I think just removing the security would be fine.

Reply:
Thanks for the clarification re: multiple selection vs multiple timelines in a single file. In that case, please consider the following suggestion:

`Select a timeline or timeline template file to import`